src/load_inputs/load_generators_data.jl

```diff
@@ -209,5 +209,275 @@ function load_generators_data!(setup::Dict, path::AbstractString, inputs_gen::Di
 			#   thus the overall is MTons/GW, and thus gen_in[g,:CO2_per_Start] is ton
 		end
 	end
+
+	load_vre_stor_data!(setup, path, inputs_gen, gen_in)
 	println(filename * " Successfully Read!")
 end
+
+
+@doc raw"""
+	check_vre_stor_validity(df::DataFrame)
+
+Function for checking that no other technology flags have been activated and specific data inputs
+	have been zeroed for the co-located VRE-STOR module
+"""
+function check_vre_stor_validity(df::DataFrame)
+	# Determine if any VRE-STOR resources exist
+	vre_stor = is_nonzero(df, :VRE_STOR)
+	r_id = df[:, :R_ID]
+
+	error_strings = String[]
+
+	function error_feedback(data::Vector{Int}, col::Symbol)::String
+		string("Generators ", data, ", marked as VRE-STOR, have ", col, " ≠ 0. ", col, " must be 0.")
+	end
+
+	function check_any_nonzero_with_vre_stor!(error_strings::Vector{String}, df::DataFrame, col::Symbol)
+		check = vre_stor .& is_nonzero(df, col)
+		if any(check)
+			e = error_feedback(r_id[check], col)
+			push!(error_strings, e)
+		end
+	end
+
+	# Confirm that any other flags/inputs are not activated (all other flags should be activated in the vre_stor_data.csv)
+	check_any_nonzero_with_vre_stor!(error_strings, df, :STOR)
+	check_any_nonzero_with_vre_stor!(error_strings, df, :THERM)
+	check_any_nonzero_with_vre_stor!(error_strings, df, :FLEX)
+	check_any_nonzero_with_vre_stor!(error_strings, df, :HYDRO)
+	check_any_nonzero_with_vre_stor!(error_strings, df, :VRE)
+	check_any_nonzero_with_vre_stor!(error_strings, df, :MUST_RUN)
+	check_any_nonzero_with_vre_stor!(error_strings, df, :LDS)
```

Comment:
Since v0.3.6 I've added a more generic check against mutually-exclusive flags. Instead of these lines, just add `:VRE_STOR` to the list of mutually-exclusive resources in `src/model/resources/resources.jl`.

Reply:
should be fixed in most recent commit